### ISSUE TITLE
How do I stop the server from code?

### ISSUE THREAD

**Issue by michaelm-rsi** [typicode/json-server#1185]
First, I'm really enjoying this project.  I'm using it with a test suite.  I can start the server with middleware as needed just fine, but I need to stop it as well at the conclusion of the test. Is there already a way to do this, and if not could we get a server.close() method?

**Comment by michaelm-rsi** [comment]
I've solved this.  I'll clone the repo, update the readme.md and issue a pull request. Again, thank you for this wonderful tool.

**Comment by faithdwsn** [comment]
Yes, I was wondering that too, how do you stop the server once you start it? I could not find that command any where in the instructions? In npm server they have CTRL  + C. But I don't see any keyboard commands for the json server. Please advise. Thank you.

**Comment by michaelm-rsi** [comment]
I issued a pull request but it wasn't merged in for some reason.  Here's the diff so you can read over what I learned.

https://github.com/typicode/json-server/pull/1186/commits/74f265bacdb52457526b1ae4dc9b76870962dc35
